### ISSUE TITLE
[Bug]: Sourcemaps are not working anymore after updating from 1.10 to 1.2.X

### ISSUE THREAD

**Issue by supercairos** [aspect-build/rules_ts#318]
### What happened?

If you update from:
```
http_archive(
    name = "aspect_rules_ts",
    sha256 = "617cc11d2ae4fe64218323da1c3776e7d25f9f45d88c1addda675d7ad736f683",
    strip_prefix = "rules_ts-1.1.0",
    url = "https://github.com/aspect-build/rules_ts/archive/refs/tags/v1.1.0.tar.gz",
)
```

To:
```
http_archive(
    name = "aspect_rules_ts",
    sha256 = "acb20a4e41295d07441fa940c8da9fd02f8637391fd74a14300586a3ee244d59",
    strip_prefix = "rules_ts-1.2.0",
    url = "https://github.com/aspect-build/rules_ts/archive/refs/tags/v1.2.0.tar.gz",
)
```
(I also tested with 1.3.0 but I still have the issue)

Source maps are wrong in comment at end of every js (build from ts) files:
Source map should look like this:
```
//# sourceMappingURL=application.js.map
```

But looks like this:
```
//# sourceMappingURL=../../../hydra-ui/__st_outdir__/__st_outdir__/hydra-ui/src/application.js.map
```

### Version

Bazel 6

```
http_archive(
    name = "rules_nodejs",
    sha256 = "08337d4fffc78f7fe648a93be12ea2fc4e8eb9795a4e6aa48595b66b34555626",
    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/5.8.0/rules_nodejs-core-5.8.0.tar.gz"],
)

http_archive(
    name = "aspect_rules_js",
    sha256 = "b3012692407d5eb2509cebdca2ece6eb0300885514e4034c20c2882c0e13c39d",
    strip_prefix = "rules_js-1.18.0",
    url = "https://github.com/aspect-build/rules_js/releases/download/v1.18.0/rules_js-v1.18.0.tar.gz",
)

http_archive(
    name = "aspect_rules_ts",
    sha256 = "acb20a4e41295d07441fa940c8da9fd02f8637391fd74a14300586a3ee244d59",
    strip_prefix = "rules_ts-1.2.0",
    url = "https://github.com/aspect-build/rules_ts/archive/refs/tags/v1.2.0.tar.gz",
)
```

TsConfig:
```
{
  "compilerOptions": {
    "allowJs": false,
    "declaration": true,
    "declarationMap": true,
    "emitDeclarationOnly": false,
    "emitDecoratorMetadata": true,
    "esModuleInterop": true,
    "experimentalDecorators": true,
    "module": "CommonJS",
    "sourceMap": true,
    "sourceRoot": "./hydra-ui",
    "strict": true,
    "target": "es6",
    "types": [
      "node",
      "jest"
    ]
  }
}

```

### How to reproduce

```shell
Check end of file source_map informations
```

### Fund our work

- [ ] Sponsor our open source work by donating a [bug bounty](https://opencollective.com/aspect-build/)

**Comment by gregmagolan** [comment]
@thesayyn is this case already by fixed by https://github.com/aspect-build/rules_ts/pull/311#event-8516435186?

**Comment by thesayyn** [comment]
nope but will be fixed by #323
